devicetypes/technisat/technisat-series-switch.src/technisat-series-switch.groovy

```diff
@@ -0,0 +1,440 @@
+/**
+ *	Copyright 2021 TechniSat
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata  {
+	definition (name: "TechniSat Series switch", namespace: "TechniSat", author: "TechniSat", vid:"generic-switch-power-energy",
```

Comment:
capitalization of your `namespace` needs to match the capitalization of the folder under `devicetypes` I believe

Reply:
This is the 4th DTH I commit with this namespace. The folder name was auto-created with the first DTH. It seems to work for the other 3, already approved, DTHs. But if it is really a problem I have to change the other DTHs as well

Reply:
After checking some other DTHs is also noticed, the namespace not always matches the folder name. For example in folder  fibargroup the namespace "FibarGroup" is used, or in drzwave the namespace "DrZWave is used"